testcases/OpTestLPM.py

```diff
@@ -99,6 +99,76 @@ def lpm_setup(self):
             if "inoperative" in str(rc):
                 raise OpTestError("LPM cannot continue as some of rsct services are not active")
 
+    def check_dmesg_errors(self, output_file=None):
+        skip_errors = ['uevent: failed to send synthetic uevent',
+                       'failed to send uevent',
+                       'registration failed',
+                       'Power-on or device reset occurred',
+                       'mobility: Failed lookup: phandle']
+
+        self.collect_errors_by_level(output_file=output_file, skip_errors=skip_errors)
+
+    def collect_errors_by_level(self, output_file=None, level_check=5, skip_errors=None):
+        """
+        Verify dmesg having severity level of OS issue(s).
+
+        :param output_file: The file used to save dmesg
+        :type output_file: str
+        :param level_check: level of severity of issues to be checked
+                            1 - emerg
+                            2 - emerg,alert
+                            3 - emerg,alert,crit
+                            4 - emerg,alert,crit,err
+                            5 - emerg,alert,crit,err,warn
+        :type level_check: int
+        :skip_errors: list of dmesg error messages which want skip
+        :type skip_errors: list
+        """
+        if not isinstance(level_check, int):
```

Comment:
why not handle this in code irrespective of user input (tets case input ) so we can avoid op-test error out , even please consider this to moving in common utils itself

Reply:
How should we handle this?

Reply:
just doing type casting to int can be done right? like  `int(level_check)` which should always return

Reply:
But that won't work if it is a string that doesn't contain a digit or a list, right?

Reply:
I say that is never the case right, either it can be int, string having a decimal value like '10' or float right

Reply:
Done